Cargo.toml

```diff
@@ -31,10 +31,11 @@ actix-session = { version = "0.9", features = [
 tokio = { version = "1.37", optional = true, features = ["macros"] }
 strum = { version = "0.26.2", features = ["derive"] }
 trait-set = "0.3.0"
+leptos-use = "0.10.10"
 
 [package.metadata.leptos-i18n]
 default = "en"
-locales = ["en", "fr"]
+locales = ["en"]
```

Comment:
Trying not to worry too much about translations at the moment.

Reply:
Make sure there's either TODO comment, or an issue for translations.